### PR TITLE
LPS-138945 Apply Workflow Instance Tracker to instance items on Metrics

### PR DESCRIPTION
**Pull request by naraandrade**
Story: [LPS-138937](https://issues.liferay.com/browse/LPS-138937)

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/java/com/liferay/portal/workflow/metrics/web/internal/configuration/FFWorkflowMetricsConfigurationActivator.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * @author Nara Andrade
+ */
+@Component(
+	configurationPid = "com.liferay.portal.workflow.metrics.web.internal.configuration.FFWorkflowMetricsConfiguration",
+	immediate = true, service = FFWorkflowMetricsConfigurationActivator.class
+)
+public class FFWorkflowMetricsConfigurationActivator {
+
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/java/com/liferay/portal/workflow/metrics/web/internal/configuration/FFWorkflowMetricsConfigurationActivator.java:33]: Hey @naraandrade, Can you please rename it to `FFWorkflowMetricsConfigurationUtil`? Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

**Comment by MarinhoFeliphe** [comment]
resent here #528

**Pull request closed without merging**